### PR TITLE
Add the interactive job type

### PR DESCRIPTION
**Pull request by johnworth**

### CODE REVIEW

File: src/main/data/99_version.sql [hunk 1]
```diff
@@ -95,3 +95,4 @@ INSERT INTO version (version) VALUES ('2.21.0:20180419.01');
 INSERT INTO version (version) VALUES ('2.21.0:20180426.01');
 INSERT INTO version (version) VALUES ('2.21.0:20180515.01');
 INSERT INTO version (version) VALUES ('2.22.0:20180523.01');
+INSERT INTO version (version) VALUES ('2.23.0:20180601.01');
```
Comment (psarando) [inline src/main/data/99_version.sql]: `v2.21.0` conversions were just released in prod.
Shouldn't this conversion still be `v2.22.0`?

**Comment by johnworth** [comment]
I have no idea.

File: src/main/conversions/v2.23.0/c2_23_0_2018060101.clj [hunk 2]
```diff
@@ -0,0 +1,18 @@
+(ns facepalm.c2-23-0-2018060101
```
Comment (psarando) [inline src/main/conversions/v2.23.0/c2_23_0_2018060101.clj]: To put it another way: I think this conversion should be `v2.22.0`, unless there's a reason this PR shouldn't be merged until the release after next...

**Review by psarando — CHANGES_REQUESTED** [review]
I think this conversion should be for `v2.22.0`.

**Review by psarando — APPROVED** [review]

**Review by slr71 — APPROVED** [review]
LGTM :shipit:

**Comment by slr71** [comment]
I'm going to be evil and merge these changes now. 😈

**Pull request merged**